File: maximum-69-number.js
/**
 * Given a positive integer num consisting of only digits 6 and 9,
 * returns the maximum number possible by changing at most one digit.
 * 
 * Example: 6 becomes 9 or 9 becomes 6.
 * 
 * @param {number} num - A number consisting only of digits 9 and 6
 * @returns {number} - Returns max number possible by changing at most 1 digit
 */
function maximum69Number (num) {
  let numArr = String(num).split('');
  for (let i = 0; i < numArr.length; i++) {
    if (numArr[i] !== '9') {
      numArr[i] = '9';
      break;
    }
  }
  numArr = numArr.join('');
  return Number(numArr);
}

File: src/algorithms/maximum-69-number.py
def maximum_69_number(num):
    """You are given a positive integer num consisting only of digits 6 and 9.
    Return the maximum number you can get by changing at most one digit (6 
    becomes 9, and 9 becomes 6).

    Args:
        num (int): A positive integer consisting of only 6 and 9 digits.

    Returns:
        int: Returns a modified integer.
    """
    num = [char for char in str(num)]
    print(num)
    for i in range(len(num)):
        if num[i] != "9":
            num[i] = "9"
            break
    return int("".join(num))
